pkg/util/version/version.go

```diff
@@ -77,3 +77,7 @@ func (v *Version) Eq(w *Version) bool {
 	}
 	return true
 }
+
+func (v *Version) GetMinorVersion() string {
```

Comment:
```suggestion
func (v *Version) MinorVersion() string {
```

As per https://go.dev/doc/effective_go#Getters